setup.py

```diff
@@ -25,11 +25,6 @@
                     'pyEphem', 'requests', 'pysatCDF', 'apexpy', 'aacgmv2',
                     'pysatMagVect', 'madrigalWeb', 'h5py', 'PyForecastTools']
 
-# keep pyglow out of requirements until pip installable
-# if sys.version_info[0] < 3:
-#     # TODO Remove when pyglow works in python 3
-#     install_requires.append('pyglow')
-
```

Comment:
Yay!

Reply:
Forgot to add this to the list of changes.  :D

Reply:
Wait, I think this is implemented wrong.  I think I should have just removed the if statement, not the install_requires bit.  @rstoneback?

Reply:
Ah, I think so!  I think you just remove the comments?

Reply:
Or just add it to the list above, since we don't need to append.  Updated readthedocs truncation as well.

Reply:
Updated version looks good to me.